src/toady/schema_validator.py

```
@@ -0,0 +1,642 @@
+                self._schema = cached_schema
+                self._build_type_map()
+                return cached_schema
+
+        logger.info("Fetching GitHub GraphQL schema...")
+
+        try:
+            # Use GitHub service to execute introspection query
+            process_result = self._github_service.run_gh_command([
+                "api", "graphql", "-f", f"query={self.INTROSPECTION_QUERY}"
+            ])
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)